sections/04_study.md

```diff
@@ -52,7 +52,104 @@ particularly notable in this area?*
 
 ### Protein secondary and tertiary structure
 
-*Jinbo Xu is writing this*
+Proteins play fundamental roles in all biological processes including the 
+maintenance of cellular integrity, metabolism, transcription/translation, and 
+cell-cell communication. Complete description of protein structures and 
+functions is a fundamental step towards understanding biological life and 
+also highly relevant in the development of therapeutics and drugs. Tons of 
+protein sequences have been generated, but fewer than 100,000 of them 
+have experimentally-solved structures. As a result, computational structure 
+prediction is essential for a majority number of protein sequences. However, 
+predicting protein 3D structures from sequence alone is very challenging, 
+especially when similar templates are not available. In the past decades, 
+various computational methods have been developed to predict protein 
+structure from different aspects, including prediction of secondary structure, 
+torsion angles, solvent accessibility, inter-residue contact map, disorder 
+regions and side-chain packing.
```

Comment:
Good point.  I spent a few minutes looking and didn't find anything with the right scope.  Let's not delay the merge looking for a review to cite, but we could add a TODO comment that it may be helpful to add one during revisions.